ush/config_defaults.yaml

```diff
@@ -380,6 +394,33 @@ platform:
 # WORKFLOW config parameters
 #-----------------------------
 workflow:
+  #
+  #-----------------------------------------------------------------------
+  #
+  # Unique ID for workflow run that will be set in setup.py
+  #
+  #-----------------------------------------------------------------------
+  #
+  WORKFLOW_ID: ""
+  #
```

Comment:
This used to be here in config_defaults before, but moved it to setupy so that the config files contain only variables that are likely to be changed by the user. That is why fixed file mappings were also moved to parm/. I think in this regard, HOMEdir etc could be put in one yaml file and moved to parm/ directory as well. What do you think about this?

Reply:
That makes sense to me. Would it be okay to stay here for this PR and I can open an issue about it for future work?

Reply:
Sounds good.